src/Bicep.Core/Semantics/Namespaces/SystemNamespaceType.cs

```diff
@@ -1594,7 +1594,7 @@ static IEnumerable<Decorator> GetBicepTemplateDecorators()
             {
                 yield return new DecoratorBuilder(LanguageConstants.ParameterSecurePropertyName)
                     .WithDescription("Makes the parameter a secure parameter.")
-                    .WithFlags(FunctionFlags.ParameterOrTypeDecorator)
```

Comment:
Since this is experimental functionality, I think we'll need to add a feature flag to Bicep to enable it - otherwise it'll be exposed to customers.

Try following the code for https://github.com/Azure/bicep/blob/aa4bdb71d9febdcd2d1e93e7ebca4deff2929e88/src/Bicep.Core/Features/IFeatureProvider.cs#L35C1-L36C46 for an example. Happy to also share some pointers offline.